lib/timers.js

```diff
@@ -462,16 +466,17 @@ function ontimeout(timer) {
   var args = timer._timerArgs;
   if (typeof timer._onTimeout !== 'function')
     return promiseResolve(timer._onTimeout, args[0]);
+  const start = TimerWrap.now();
   if (!args)
     timer._onTimeout();
   else
     Reflect.apply(timer._onTimeout, timer, args);
   if (timer._repeat)
-    rearm(timer);
+    rearm(timer, start);
```

Comment:
Why is `TimerWrap.now()` not called here? It is the only used spot in the function, so it does not have to be called if `timer._repeat` is falsy.